packages/typespec-ts/src/modular/helpers/operationHelpers.ts

```diff
@@ -583,7 +664,46 @@ function deserializeResponseValue(type: Type, restValue: string): string {
       } else {
         return restValue;
       }
+    case "byte-array":
+      utilImports.add("stringToUint8Array");
```

Comment:
@xirzec  we will use the utilImports to add the helper function names here. And this function deserializeResponseValue(type: Type, restValue: string) doesn't take a parameter to add imports, and it's being called with a long chain, if we use utilImports as the parameter, we will need to change those functions all the way up. which I am not sure if it's worth it.